ts/blsSigner.ts

```diff
@@ -0,0 +1,60 @@
+import {
+    solG2,
+    Domain,
+    getPubkey,
+    g2ToHex,
+    sign,
+    g1ToHex,
+    aggregateRaw,
+    mclG1,
+    solG1,
+    SecretKey,
+    randFr,
+    PublicKey
+} from "./mcl";
+
+export interface SignatureInterface {
+    mcl: mclG1;
```

Comment:
keeping mclG1 here so that `aggregate` can work on it directly without converting solG1 back to mclG1.